crates/agentgateway/proto/resource.proto

```diff
@@ -434,7 +434,31 @@ message FrontendPolicySpec {
     Fields fields = 2;
   }
 
-  message Tracing {}
+  message Tracing {
+    BackendReference provider_backend = 1;
+    repeated TracingAttribute attributes = 2;
+    repeated TracingAttribute resources = 3;
+      // When true, use plaintext (http) for OTLP endpoint; otherwise use TLS (https).
+      // If unset, defaults to true to preserve existing behavior.
+      optional bool insecure = 4;
```

Comment:
This should be a property of the Backend policies?

Reply:
Ah okay so then our default behavior is insecure since we just use the backendRef with no backend policy:
```
  frontend:
    tracing:
      backendRef:
        name: opentelemetry-collector
        namespace: default
        port: 4317
      protocol: GRPC
      clientSampling: "true"
      randomSampling: "true"
      attributes:
        add:
          - expression: 'request.headers["user-agent"]'
            name: http.useragent
```

Reply:
Yep. Well, technically if its in the mesh we can do auto mesh TLS to it! So its not insecure.